src/main/java/mekanism/common/tile/TileEntityBin.java

```diff
@@ -121,6 +121,25 @@ public InteractionResult onRightClick(Player player) {
         return InteractionResult.PASS;
     }
 
+    public boolean toggleLock() {
+        return setLocked(!binSlot.isLocked());
+    }
+
+    public boolean setLocked(boolean isLocked) {
+        if (getTier() == BinTier.CREATIVE) {
+            return false;
+        }
+        final boolean changed = binSlot.setLocked(isLocked, false);
+        if (changed) {
```

Comment:
This boolean can just be inlined and have a simple `return true` statement at the end of the if branch and change the current return statement to `return false`